app/Http/Controllers/ProductController.php

```diff
@@ -7,6 +7,7 @@
 use Illuminate\Http\JsonResponse;
 use Illuminate\Http\Request;
 use Illuminate\Support\Facades\Validator;
+use Illuminate\View\View;
```

Comment:
View?

Reply:
it is for annotation
I changed the show method to return a view when the /product/{id} endpoint is accessed